pom.xml

```diff
@@ -64,6 +64,11 @@
     <dependencyManagement>
         <dependencies>
             <!-- Carbon Kernel dependencies -->
+            <dependency>
+                <groupId>com.sun.org.apache</groupId>
+                <artifactId>jaxp-ri</artifactId>
+                <version>1.4</version>
```

Comment:
Best practice is to add the dependencies in the parent pom under dependency management and use in the child pom. And use a property to specify the version